data/segments.json

```diff
@@ -0,0 +1,39 @@
+{
+  "newsletterUserActive": "true",
```

Comment:
Change these to `true` (no quotes) and change the templates to `if $data.newsletterUserActive` etc.